lib/extension/bridge.ts

```diff
@@ -500,6 +501,23 @@ export default class Bridge extends Extension {
         }, null);
     }
 
+    @bind async deviceReinterview(message: string | KeyValue): Promise<MQTTResponse> {
+        if (typeof message !== 'object' || !message.hasOwnProperty('id')) {
+            throw new Error(`Invalid payload`);
+        }
+
+        const device = this.zigbee.resolveEntityAndEndpoint(message.id).entity as Device;
+        if (!device) throw new Error(`Device '${message.id}' does not exist`);
+
+        try {
+            await device.zh.interview();
+        } catch (error) {
+            throw new Error(`re-interview of '${device.name}' (${device.ieeeAddr}) failed: ${error}`, {cause: error});
+        }
+
```

Comment:
```suggestion
await this.publishDevices()
```

This should update the frontend after the interview.